client/src/components/friends-page/FriendsContent.css

```diff
@@ -56,3 +61,29 @@
   position: fixed;
   z-index: 100;
 }
+
+.names-left {
+  font-family: "Cookie", cursive;
+  font-size: 41px;
+  color: var(--deep-blue);
+  position: absolute;
+  left: 100px;
+  bottom: 30px;
```

Comment:
For example here its from inside to outside, which i think is fine. I would just stick with one sorting through the rulesets